analytics/spark/src/main/java/mil/nga/giat/geowave/analytic/spark/sparksql/udf/wkt/WKTGeomFunction.java

```diff
@@ -5,6 +5,8 @@
 import org.slf4j.LoggerFactory;
 
 import com.vividsolutions.jts.geom.Geometry;
+import com.vividsolutions.jts.geom.prep.PreparedGeometry;
```

Comment:
added these prepared geometry imports but seemingly never use them (this is the case it seems in a bunch of the classes)